src/vote.rs

```diff
@@ -39,6 +40,13 @@ pub struct DkgSignedVote {
     pub sig: Signature,
 }
 
+// Manual impl of Debug to skip vote content details
+impl fmt::Debug for DkgSignedVote {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        write!(f, "{:?}:{:?}", self.voter, self.vote)
+    }
+}
+
```

Comment:
Maybe move to the debug module as well?

Reply:
I initially wanted to but `self.vote` is a private field and making it public just for debug feels off 🤔